### ISSUE TITLE
TTL supports the property value of  `NULL`, but the TTL function does not take effect

### ISSUE THREAD

**Issue by abby-cyber** [vesoft-inc/nebula#4693]
<img width="698" alt="image" src="https://user-images.githubusercontent.com/78209557/194686824-19ebd8fd-ead4-4cb9-9c44-cee39c4f50d3.png">

No `start_time` property was inserted, in which case the default value of `start_time` is `NULL`.

<img width="857" alt="image" src="https://user-images.githubusercontent.com/78209557/194686841-c688fea3-a858-4a4e-9fa2-09c52f232e51.png">

TTL did not take effect after a while

<img width="948" alt="image" src="https://user-images.githubusercontent.com/78209557/194686912-559f1d62-05ba-4acd-bc14-a67490cbb498.png">

**Comment by HarrisChu** [comment]
![image](https://user-images.githubusercontent.com/1726587/205622072-bb54c53b-e21f-4608-ad78-4f2bdd09208e.png)

**Comment by abby-cyber** [comment]
> ![image](https://user-images.githubusercontent.com/1726587/205622072-bb54c53b-e21f-4608-ad78-4f2bdd09208e.png)

No, you cannot just paste the words instead of doing nothing. What we need to do is to fix or enhance this feature. Plus, the sentence in the doc was added by me after I submitted this issue.
